tests/fixtures/datasets.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from pathlib import Path
+
+
+@pytest.fixture
+def sleap_data_dir(pytestconfig):
+    """Dir path to sleap data."""
+    return Path(pytestconfig.rootdir) / "tests/data"
+
+
+@pytest.fixture
+def fly32(sleap_data_dir):
+    """Sleap single fly .slp and video file paths."""
+    return Path(sleap_data_dir) / "fly32.pkg.slp"
```

Comment:
Definitely use a smaller fixture like the ones in: https://github.com/talmolab/sleap-io/tree/main/tests/data/slp